modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/display/context/FDSViewsDisplayContext.java

```diff
@@ -93,13 +104,36 @@ public String getFDSViewsURL(String fdsEntryId, String fdsEntryLabel) {
 	public String getFDSViewURL() {
 		return PortletURLBuilder.create(
 			PortletURLFactoryUtil.create(
-				_portletRequest, FDSViewsPortletKeys.FDS_VIEWS,
-				PortletRequest.RENDER_PHASE)
+				_renderRequest, FDSViewsPortletKeys.FDS_VIEWS,
+				RenderRequest.RENDER_PHASE)
 		).setMVCPath(
 			"/fds_view.jsp"
 		).buildString();
 	}
 
+	public String getPermissionsURL() {
```

Comment:
This should probably be `getPermissionsURL(ObjectDefinition objectDefinition)`. Later on, we will have permissions for FDSViews.